### PR TITLE
specify code lifetime is 60 seconds in api doc

### PR DESCRIPTION
**Pull request by mystiberry-okta**
## Description:
- Already in authn guide, but api docs need to include limit as well. 

### Resolves:
[OKTA-148500](https://oktainc.atlassian.net/browse/OKTA-148500)

### CODE REVIEW

**Comment by mystiberry-okta** [comment]
@johngronberg-okta I left the previous (already published) commit in this PR so you could look at it. Just want to make all the changes are correct. Thanks!

File: _source/_docs/api/resources/apps.md [hunk 1]
```diff
@@ -1007,6 +1007,8 @@ Adds an OAuth 2.0 client application. This application is only available to the
 
 * All redirect URIs must be absolute URIs and must not include a fragment component.
 
+* `authorization_code` has a lifetime of 60 seconds and can't be configured. Okta reserves the right to change this value in the future.
```
Comment (johngronberg-okta) [inline _source/_docs/api/resources/apps.md]: I don't think we need to add this line.

Comment (mystiberry-okta) [reply _source/_docs/api/resources/apps.md]: Ack, will remove the 2nd sentence (or the whole thing?).

Comment (johngronberg-okta) [reply _source/_docs/api/resources/apps.md]: The whole line. It's not really relevant to app setup.

Comment (mystiberry-okta) [reply _source/_docs/api/resources/apps.md]: Ack.

**Comment by mystiberry-okta** [comment]
Ready for re-review.

**Pull request merged**